src/pages/Edital/Inscricao/Components/ModalProducao.tsx

```diff
@@ -0,0 +1,243 @@
+                  variant: "h4",
+                  p: 1,
+                }}
+                sx={{ px: 3 }}
+                // subheader="O qualis das publicações será o mais atual aprovado pela CAPES."
+                // subheaderTypographyProps={{ align: "center" }}
+              />
+              <Divider sx={{ mx: 3 }} />
+
+              <form
+                id="add-producao-form"
+                onChange={handleFormChange}
+                // onSubmit={sendForm}
+              >
+                <CardContent sx={{ px: 5 }}>
+                  <Grid
+                    container
+                    direction="column"
+                    justifyContent="center"
+                    alignItems="center"
+                    sx={{ width: "100%" }}
+                  >
+                    <Grid
+                      container
+                      direction="row"
+                      justifyContent="space-between"
+                    >
+                      <Grid item xs={10}>
+                        <FormControl fullWidth>
+                          <InputLabel id="select-categoria">
+                            Categoria
+                          </InputLabel>
+                          <Select
+                            labelId="select-categoria"
+                            label="Categoria"
+                            id="demo-simple-select"
+                            value={notaCategoria}
+                            onChange={handleChange}
+                          >
+                            <MenuItem value="0" disabled>
+                              Selecione uma categoria
+                            </MenuItem>
+                            {edital?.categorias_producao.map((categoria) => (
+                              <MenuItem
+                                key={categoria.id}
+                                value={categoria.pontuacao}
+                              >
+                                {categoria.nome}
+                              </MenuItem>
+                            ))}
+                          </Select>
+                        </FormControl>
+                      </Grid>
+                      <Grid item xs={1.7}>
+                        <FormControl>
+                          <InputLabel htmlFor="Nota">Nota</InputLabel>
+                          <OutlinedInput
+                            id="nota_categoria"
+                            name="nota_categoria"
+                            label="Nota"
+                            type="text"
+                            value={notaCategoria}
+                            inputProps={{ readOnly: true }}
+                          />
+                        </FormControl>
+                      </Grid>
+                    </Grid>
+
+                    <FormControl required fullWidth margin="normal">
+                      <AttachInput
+                        inputName="producao_cientifica_file"
```

Comment:
Esse name aqui é oq ele vai usar pra saber qual propriedade do producaoData que ele vai usar no evento de change

Ou vc dá outro nome aqui no inputName
Ou vc muda o nome da propriedade lá no producaoData

(ou os 2 rs, n gostei de nenhum dos 2 nomes rs)

Reply:
N sei se ficou claro, mas tem q ser igual

Reply:
sim, mas dentro do setProducaoCientifica eu n usei o target.name
usei diretamente files

tbm n gostei dos nomes, to aberto a sugestoes

Reply:
aaaaaah, vdd